robot_sort/robot_sort.py

```diff
@@ -96,15 +96,45 @@ def sort(self):
         """
         Sort the robot's list.
         """
-        # Fill this out
-        pass
 
+        # swap None for 0th index item
+        self.swap_item() 
 
-if __name__ == "__main__":
+        while True:
+
```

Comment:
Brilliant use of selection sort; this is the classic implementation. great work!